ui/src/app/applications/components/application-details/application-details.scss

```diff
@@ -344,4 +348,24 @@ $header: 120px;
 .sb-page-wrapper .top-bar.row {
     display: none !important;
 }
-}
\ No newline at end of file
+}
+
+.resource-parent-node-info {
```

Comment:
This looks better than the demo you showed me yesterday.  The one thing I would suggest is to differentiate between the 'label' or 'name' of the piece of information, versus, its value.  Recall the Sync Status panel I mentioned, where it shows the Author and Comment.  The value is slightly bolded.  I think if you change it to look like that, it will add to the fit-and-finish and make it consistent with at least that part of the UI. Note also the vertical spacing between lines.

Reply:
How does this look? Label name and value has different font-weight and can identify distinctly.
![Screenshot 2023-03-29 at 9 15 27 AM](https://user-images.githubusercontent.com/11219262/228602078-a4ee6660-24ea-4838-88c0-bbab8313f6fa.png)